import "msgFile.proto";

message MessageProto {
  required int32 cmdType = 1;  
  
  repeated MsgFile msgFileData = 2;
}